### DISCUSSION TITLE
Is is possible to link app docs and its sub app docs in swagger UI using FastAPI routines?

### DISCUSSION THREAD

**Discussion by poul1x** [fastapi/fastapi#6739] [Questions] [answered]
I want my app to have a hyperlink from root `/docs` to sub app `/docs`.
Is is possible to link app docs and its sub app docs in swagger UI using FastAPI routines?

**Comment by Kludex** [comment]
You mean merging both? There's no native way to do that. 

You'll need to create a custom openapi. 

I'm not on my computer, so I'll not provide a link, but you can search for "custom openapi FastAPI"

**Comment by poul1x** [comment]
I mean, in `/docs` of main app I have a hyperlink to `/docs` of sub app. In swagger UI

**Comment by waynerv** [comment, marked as answer]
Seems unlikely to happen for swagger UI not providing a such feature.

Maybe you can custom the openapi and write the hyperlink in `description`:
```
openapi_schema = get_openapi(
        title=settings.PROJECT_NAME,
        version='0.0.1',
        description='leave hyperlink here',
        routes=app.routes,
    )
```

**Comment by tiangolo** [comment]
Thanks for the help here @Kludex and @waynerv ! 👏  🙇 

Thanks for coming back and closing the issue @poul1x  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
